ledger/internal/jet/utils.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Let's name this file "bits".
https://github.com/andreyromancev/go-conventions#naming